lib/jsdom/living/interfaces.js

```diff
@@ -184,9 +185,15 @@ function install(window, name, interfaceConstructor) {
 }
 
 exports.installInterfaces = function (window) {
+  // TODO: Fix me
+  Object.defineProperty(window, "Error", {
```

Comment:
This line needs to be updated once we settle on an approach to expose ECMAScript globals (https://github.com/jsdom/jsdom/issues/2727). This is a tempory solution to make the tests pass.